pp/integrations/plugin-packs/procedures/applications-trendmicro-iwsva-snmp.md

```diff
@@ -2,48 +2,635 @@
+
+| Service Alias     | Service Template                                   | Service Description                         | Discovery  |
+|:------------------|:---------------------------------------------------|:--------------------------------------------|:----------:|
+| Cpu               | App-Trendmicro-Iwsva-Cpu-SNMP-custom               | Check CPU usage                             |            |
+| Disk-Global       | App-Trendmicro-Iwsva-Disk-Global-SNMP-custom       | Check the rate of free space on disks       | X          |
+| Load              | App-Trendmicro-Iwsva-Load-SNMP-custom              | Check the server load average               |            |
+| Memory            | App-Trendmicro-Iwsva-Memory-SNMP-custom            | Check the rate of the utilization of memory |            |
+| Proxy-Connections | App-Trendmicro-Iwsva-Proxy-Connections-SNMP-custom | Check proxy connections usage               |            |
+| Swap              | App-Trendmicro-Iwsva-Swap-SNMP-custom              | Check virtual memory usage                  |            |
+| Traffic-Global    | App-Trendmicro-Iwsva-Traffic-Global-SNMP-custom    | Check the bandwidth of interfaces           | X          |
+
+> The services listed above are created automatically when the **App-Trendmicro-Iwsva-SNMP-custom** host template is used.
+
+> If **Discovery** is checked, it means a service discovery rule exists for this service template.
+
+</TabItem>
+</Tabs>
+
+### Discovery rules
+
+#### Host discovery
+
+| Rule name       | Description                                                                                                                                                                                                                                              |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **App-Trendmicro-Iwsva-SNMP-custom** host template |
+
+More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).
+
+#### Service discovery
+
+| Rule name                              | Description                                                   |
+|:---------------------------------------|:--------------------------------------------------------------|
+| App-Trendmicro-Iwsva-SNMP-Disk-Name    | Discover the disk partitions and monitor space occupation     |
+| App-Trendmicro-Iwsva-SNMP-Traffic-Name | Discover network interfaces and monitor bandwidth utilization |
```

Comment:
FR has status too

Reply:
Indeed because of `--add-status` [option](https://github.com/centreon/centreon-plugins/blob/develop/src/snmp_standard/mode/interfaces.pm#L1631) in the pack